ui/src/app/components/add-index-form/add-index-form.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, OnInit } from '@angular/core'
+import { FormArray, FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms'
+import IConv, { ICreateIndex } from 'src/app/model/Conv'
+import { DataService } from 'src/app/services/data/data.service'
+
+@Component({
+  selector: 'app-add-index-form',
+  templateUrl: './add-index-form.component.html',
+  styleUrls: ['./add-index-form.component.scss'],
+})
+export class AddIndexFormComponent implements OnInit {
+  addIndexForm: FormGroup
+  tableNames: string[] = []
+  currentColumns: string[] = []
+  conv: IConv = {} as IConv
+  constructor(private fb: FormBuilder, private data: DataService) {
+    this.addIndexForm = this.fb.group({
+      tableName: ['', Validators.required],
+      indexName: ['', Validators.required],
+      ColsArray: this.fb.array([]),
+    })
+  }
+
+  ngOnInit(): void {
+    this.data.conv.subscribe({
+      next: (res: IConv) => {
+        this.conv = res
+        this.tableNames = Object.keys(res.SpSchema)
+      },
+    })
+  }
+
+  get ColsArray() {
+    return this.addIndexForm.controls['ColsArray'] as FormArray
+  }
+
+  selectedTableChage(tableName: string) {
+    console.log(tableName)
```

Comment:
Remove extra console logs if the development of component is complete so that we don't checkin the code with unnecessary console logs

Reply:
Removed console.logs. addressed in PR 287